src/routes/portfolio/mappers/portfolio.mapper.spec.ts

```diff
@@ -0,0 +1 @@
+it.todo('PortfolioMapper');
```

Comment:
The mapping is being reference in local development. After confirmed, tests will be added and this marked for review.